AUTHORS

```diff
@@ -0,0 +1,45 @@
+Anindya Ghosh <anindya128@yahoo.com>
+Antoine Grimaldi <antoine.grimaldi@univ-amu.fr>
+AntoineGrimaldi <antoine.grimaldi@univ-amu.fr>
+Carsten Nielsen <carstenlau.nielsen@uzh.ch>
+Fabrizio Ottati <45739782+fabhertz95@users.noreply.github.com>
+Fabrizio Ottati <fabriziottati@gmail.com>
```

Comment:
I'm not sure why this file hasn't been excluded in the gitignore already, could you please remove this file and ChangeLog. I fixed the .gitignore in the current develop branch